src/main/java/uk/gov/ch/util/Constants.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ch.util;
+
+public class Constants {
+
+    private Constants() {
+    }
+
+    public static final String OVERSEAS_ENTITY_ID_KEY = "overseas_entity_id";
+    public static final String NOT_FOUND_MESSAGE = "Contact email not found for overseas entity: ";
```

Comment:
This seems to be used in only one place, so would suggest just defining it as a private static constant in the `OverseasOverseasEntityDataServiceImpl` class.